### PR TITLE
MT-261

### PR DESCRIPTION
**Pull request by MeenuDhanoa**

### CODE REVIEW

File: src/components/CRUD.js [hunk 1]
```diff
@@ -280,6 +297,16 @@ class CRUD extends React.Component {
                   actions.insertImage,
                   actions.insertVideo,
                 ]}
+                iconMap={{
+                  [actions.setBold]: ({ tintColor }) => (<>{setBoldSvg({ width: 25, height: 25, fill: tintColor })}</>),
+                  [actions.setItalic]: ({ tintColor }) => (<>{setItalicSvg({ width: 25, height: 25, fill: tintColor })}</>),
```
Comment (csillaj) [inline src/components/CRUD.js:302]: why is this needed? I think the icons have an "on" state when clicked even without this mapping

**Review by DidierF — APPROVED** [review]

File: src/components/CRUD.js (hunk 1 above)
Comment (MeenuDhanoa) [reply src/components/CRUD.js:302]: I have added mapping to use custom icons as per design update.

**Review by csillaj — APPROVED** [review]
